docs/user/introduction.md

```diff
@@ -0,0 +1,72 @@
+* Container or VM workloads on Podman using docker-compose or Quadlets, Kubernetes services on MicroShift using kustomize or Helm.
+* Agent-based architecture, allowing:
+  * scalable and robust management under adverse networking conditions (agent "calls home" when the device has connectivity),
+  * autonomous and safe OTA updates (agent downloads and verifies assets before updating, then transactionally updates or rolls back), and
+  * timely yet resource-unintensive monitoring of devices and applications (agent notifies service on update progress and alarms).
+* A secure yet friction-free device lifecycle (enrollment, certificate rotation, attestation, and decommissioning) based on hardware root-of-trust.
+* Pluggable identity/authentication providers (initially Keycloak / Generic OIDC Providers and OpenShift Authentication API)
+* Pluggable authorization providers (initially SpiceDB and Kubernetes RBAC)
+* Pluggable certificate providers (initially built-in CA and Kubernetes Cert-Manager)
+
+## Concepts
+
+**Device** - A combination of a (real or virtual) machine, operating system, and application workload(s) that function together to serve a specific purpose.
+
+**Device Spec** - A specification of the state (e.g. configuration) the user wants a Device to have.
+
+**Device Status** - A record of the state (e.g. configuration) that the Device is reported to actually have.
+
+**Device Template** - A template for Device Specs that serves to control drift between the configurations of Devices.
+
+**Fleet** - A group of Devices governed by a common Device Template and common management policies.
+
+**Labels** - A way for users to organize their Devices and other resources, for example to record their location ("region=emea", "site=factory-berlin"), hardware type ("hw-model=jetson", "hw-generation=orin"), or purpose ("device-type=autonomous-forklift").
+
+**Label Selector** - A way for users to group or filter their devices and other resources based on their assigned labels, e.g. "all devices having 'site=factory-berlin' and 'device-type=autonomous-forklift').
+
+**Service** -  The Flight Control Service handles user and agent authentication and authorization, device enrollment and inventory, rolling out updates to devices, and rolling up status from devices.
+
+**Agent** - The Flight Control Agent runs on each device and is responsible for securely enrolling into the Service, querying the Service for updates, autonomously applying these updates, and reporting status on the updates and the health of devices back to the Service.
+
+## High-Level Architecture
+
+Flight Control consists of the highlighted components show in the following high-level architecture diagram:
+
+![no-alt-text](../images/flightctl-highlevel-architecture.svg#only-light)
+![no-alt-text](../images/flightctl-highlevel-architecture-dark.svg#only-dark)
```

Comment:
When viewing this file on github, it shows up as two separate images, one after the other.

Reply:
Thanks for catching this. Apparently, the syntax has been deprecated in favour of inline HTML.